examples/lfric/eg2/module_inline_trans.py

```diff
@@ -31,43 +31,46 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
-# Author R. Ford STFC Daresbury Lab
-# Modified by S. Siso, STFC Daresbury Laboratory
+# Author R. W. Ford, STFC Daresbury Laboratory
+# Modified by S. Siso, and A. R. Porter, STFC Daresbury Laboratory
+
+''' Example transformation script showing the use of the module-inline
+    transformation for the LFRic domain. '''
 
-''' example showing the use of the module-inline transformation '''
 from __future__ import print_function
-import os
-from psyclone.parse.algorithm import parse
-from psyclone.psyGen import PSyFactory
 from psyclone.transformations import KernelModuleInlineTrans
+from psyclone.psyGen import Kern
+
+
+def trans(psy):
+    '''
+    PSyclone transformation routine. This is an example which module-inlines
+    the kernel used in the second 'invoke' in the supplied PSy object.
 
+    :param psy: the PSy object that PSyclone has constructed for the \
+                'invoke'(s) found in the Algorithm file.
+    :type psy: :py:class:`psyclone.dynamo0p3.DynamoPSy`
 
-def inline():
-    ''' function exercising the module-inline transformation '''
-    _, info = parse(os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                                 "..", "..", "..", "src", "psyclone", "tests",
-                                 "test_files", "dynamo0p1", "algorithm",
-                                 "1_single_function.f90"),
-                    api="dynamo0.1")
-    psy = PSyFactory("dynamo0.1").create(info)
+    :returns: the transformed PSy object.
+    :rtype: :py:class:`psyclone.dynamo0p3.DynamoPSy`
+
+    '''
     invokes = psy.invokes
     print(psy.invokes.names)
-    invoke = invokes.get("invoke_0_testkern_type")
+    invoke = invokes.get("invoke_1")
     schedule = invoke.schedule
     schedule.view()
-    kern = schedule.children[0].loop_body[0]
+    # Find the kernel we want to inline.
+    kern = schedule.walk(Kern)[0]
     # setting module inline directly
```

Comment:
I would suggest updating this comment to be the same style as the others (starting with upper case and ending with a full stop).

Reply:
Done.